src/pages/components/TextCopy/TextCopy.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+interface ITextCopyProps {
+  text: string;
+}
+
+const TextCopy: React.FC<ITextCopyProps> = (props) => {
+  const [text, setText] = useState('');
+  const [isEdited, setIsEdited] = useState(false);
+
+  const updateText = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setText(e.target.value);
```

Comment:
Dá pra deixar o event explicito aqui.

```suggestion
  const updateText = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
    setText(event.target.value);
```